core/domain/suggestion_registry.py

```diff
@@ -1101,27 +1101,30 @@ class ReviewerAndSuggestionCounts(python_utils.OBJECT):
     """Domain object for the ReviewerAndSuggestionCountsModel.
 
     Attributes:
-        translation_reviewer_counts_per_lang: dict. A dictionary that contains
-            the total number of translation reviewers for each language that
-            translation suggestions are offered in.
-        translation_suggestion_counts_per_lang: dict. A dictionary that
-            contains the total total number of translation suggestions that are
-            currently in review per language.
+        translation_reviewer_counts_by_lang: dict. A dictionary where the keys
+            represent the languages that translation suggestions are offered in
+            and the values correspond to the total number of reviewers who have
+            permission to review translation suggestions in that language.
+        translation_suggestion_counts_by_lang: dict. A dictionary where the
+            keys represent the languages that translation suggestions are
+            offered in and the values correspond to the total number of
+            translation suggestions that are currently in review in that
+            language.
         question_reviewer_count: int. The total number of reviewers who have
             permission to review question suggestions.
         question_suggestion_count: int. The total number of question
             suggestions that are currently in review.
     """
 
     def __init__(
-            self, translation_reviewer_counts_per_lang,
-            translation_suggestion_counts_per_lang,
+            self, translation_reviewer_counts_by_lang,
```

Comment:
Same question -- lang or lang code? Please fix throughout this PR; the two concepts are different.

Reply:
Changed, thanks :)